src/ui/components/post/post_listings.rs

```diff
@@ -3,15 +3,15 @@ use lemmy_api_common::lemmy_db_views::structs::PostView;
 use leptos::*;
 
 #[component]
-pub fn PostListings(posts: MaybeSignal<Vec<PostView>>) -> impl IntoView {
+pub fn PostListings(
+  posts: MaybeSignal<Vec<PostView>>,
+  error: RwSignal<Option<String>>,
+) -> impl IntoView {
   view! {
-    <ul>
+    <table class="table">
```

Comment:
Why use table?

Reply:
I disagree that this is tabular data, but the markup can be changed in a later PR.